README.md

```diff
@@ -6,7 +6,7 @@ We would like to open up the design process for the did:pkh [**method specificat
 
 Regular meetings for this CCG work item will be held every two weeks on Thursdays at [1.30PM Eastern Time](https://www.timeanddate.com/worldclock/), on a 
 [workitem-specific jitsi room](https://meet.w3c-ccg.org/didpkh) hosted by the CCG.  Minutes will be published 
-the same way as other work items.  
+in a [running markdown file](agenda1.md) here in the repo.  
```

Comment:
I strongly advise against using one file/document for perpetual minute-keeping.  Dated files, one per meeting, timestamped if multiple meetings occur in one day, are the typical method, which I heartily endorse.

Reply:
Ok sounds good, will do that instead if that's the norm.